include/class.mailfetch.php

```diff
@@ -126,14 +126,18 @@ function getMaxFetch() {
         return $this->ht['max_fetch'];
     }
 
+    function getMailFolder() {
+        return $this->mailbox_encode($this->ht['folder']) ?: 'INBOX';
```

Comment:
I know mailbox_encode returns a null on null folder...  but that's not guaranteed forever should the logic get modified...  something like below will make me comfortable. 

```php
return $this->mailbox_encode($this->ht['folder'] ?: 'INBOX');
```

Reply:
Done